extmod/asyncio/stream.py

```diff
@@ -114,6 +117,18 @@ def open_connection(host, port):
         if er.errno != EINPROGRESS:
             raise er
     yield core._io_queue.queue_write(s)
+    # wrap with SSL, if requested
+    if ssl:
+        if ssl is True:
+            import ssl as _ssl
+
+            ssl = _ssl.SSLContext(_ssl.PROTOCOL_TLS_CLIENT)
+        if not server_hostname:
+            server_hostname = host
+        s = ssl.wrap_socket(s, server_hostname=server_hostname, do_handshake_on_connect=False)
+        s.setblocking(False)
+        ss = Stream(s)
+        yield core._io_queue.queue_write(s)
```

Comment:
These two lines here can probably be de-dented, and then the `ss = Stream(s)` and `yield core._io_queue.queue_write(s)` lines above (at line 113 and 119) be removed.  Then this bit here looks like:
```py
    ...
    s.setblocking(False)
ss = Stream(s)
yield core._io_queue.queue_write(s)
return ss, ss
```
(The call to wrap_socket doesn't need the socket to be connected when `do_handshake_on_connect=False`.)